src/shared/config/frontend.config.ts

```diff
@@ -1,11 +1,29 @@
-import { IsBoolean, IsInt, IsNotEmpty, IsPositive, IsString, Max, Min } from 'class-validator';
+import {
+    IsBoolean,
+    IsInt,
+    IsNotEmpty,
+    IsOptional,
+    IsPositive,
+    IsString,
+    Max,
+    Min,
+    isBoolean,
+    isInt,
+    isString,
+} from 'class-validator';
+
+import { OneOf } from '../util/one-of.validator.decorator.js';
 
 export class FrontendConfig {
     @IsInt()
     @Min(1024)
     @Max(10000)
     public readonly PORT!: number;
 
+    @IsOptional()
+    @OneOf(isString, isInt, isBoolean)
+    public readonly TRUST_PROXY?: string | number | boolean;
```

Comment:
add some example to /config/config.dev.json to understand what is expected here.

Reply:
Set an example value and also linked to the express-docs